q_it_invalid_sqs_url_error.rb

```diff
@@ -0,0 +1 @@
+class QItInvalidSQSUrlError < StandardError; end
```

Comment:
I am conflicted as to whether you can have all the errors in a single file or have separate files.
I am leaning towards a single file - we don't have anything significant in each class that warrants individual files for each class.
I am fine either way.

Reply:
Maybe we can put it in a single file. As you mentioned, we do not have anything significant in any of the error classes. Additionally having them in a single file will make it easier to manage in case of any changes in the future(which I see very less possibility of). So will move them all to a single file.